cterasdk/core/users.py

```diff
@@ -90,3 +110,11 @@ def delete(self, name):
         logging.getLogger().info('User deleted. %s', {'user': name})
 
         return response
+
+
+UserAccount = namedtuple('UserAccount', ('directory', 'name'))
```

Comment:
I think that because of the "local" option, it will be better if this is an actual class instead of a named tuple. The __init__ method should have an optional field for the directory and if it is not passed, the user is local. It should then expose a property of is_local.
This way you eliminate the bugs when misspelling "local" (when setting or when checking)

Reply:
Transitioned to a UserAccount class. Again, I held off on updating the code until we import all namedtuples and classes to the client's context